lms/lti.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+
+@dataclass
+class _ParamMapping:
+    key: str
+    """Name of the parameter in LTI1.3"""
+
+    sub: Optional[str] = None
+    """For nested values, name in the second object"""
+
+    function: Optional[Callable] = None
```

Comment:
I had this as "method" before but the resulting method needed to be a static one for pylint and having a function is a bit nicer as it doesn't need a `gettattr` as the method approach.